go.sum

```diff
@@ -1,6 +1,10 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/checkpoint-restore/go-criu v0.0.0-20191125063657-fcdcd07065c5 h1:950diauOSoCNaQfvLE0WaFadyI/ilKIjb6jEUQnm+hE=
 github.com/checkpoint-restore/go-criu v0.0.0-20191125063657-fcdcd07065c5/go.mod h1:TrMrLQfeENAPYPRsJuq3jsqdlRh3lvi6trTZJG8+tho=
+github.com/checkpoint-restore/go-criu v0.0.0-20200515115155-5fe4432ec9c4 h1:HAWVG+00NF6SNU9g16Epd/46Iq6QLBWWiH+SoegMtfU=
+github.com/checkpoint-restore/go-criu v0.0.0-20200515115155-5fe4432ec9c4/go.mod h1:TrMrLQfeENAPYPRsJuq3jsqdlRh3lvi6trTZJG8+tho=
```

Comment:
AFAICS these lines should not be there (and yeah I know that `go mod` is tough).

Maybe try running `go mod tidy`?

Reply:
@adrianreber PTAL ^^^

Reply:
I removed it manually as `go mod tidy` removed a couple more lines.